saleor/dashboard/page/forms.py

```diff
@@ -0,0 +1,21 @@
+from django import forms
+
+from ...page.models import Page
+from ..product.forms import RichTextField
+
+
+class PageForm(forms.ModelForm):
+
+    class Meta:
+        model = Page
+        exclude = []
+
+    content = RichTextField()
+
+    def clean_url(self):
+        """
+        Make sure url is not being written to database with uppercase.
```

Comment:
You're using a docstring here, while this comment looks more like a simple inline comment.